notifications/templates/email/informal_update/informal_pdf.html

```diff
@@ -0,0 +1,109 @@
+        font-weight: bold;
+        font-size: 24px;
+        text-align: center;
+      }
+      .heading {
+        font-size: 20px;
+      }
+      .sub-heading {
+        font-size: 15px;
+        font-weight: bold;
+      }
+      .desc {
+        font-size: 15px;
+      }
+      .actions {
+          font-size: 13px;
+          font-weight: bold;
+          padding-left: 20px;
+      }
+    .img-informal {
+            width: 120px;
+            height: 120px;
+        }
+
+    </style>
+
+    <body>
+        <div class="img text-left">
+            <img src="https://dsgoproxyapp.azurewebsites.net/assets/graphics/layout/go-logo-2020.png" alt="Logo: International Federation of Red Cross and Red Crescent Societies" height="40" border="0" style="-ms-interpolation-mode:bicubic;" />
+        </div>
+        <h1 class="title-heading-main">{{title}}</h1>
+        <h2 class="heading">Situational Overview</h2>
+        <p class="desc"> {{situational_overview}}</p>
+
+        <h2 class="heading">Graphics</h2>
+        <div class="d-flex p-2">
+        {% for map in map_list %}
+            <img class = 'img-informal' src="http://{{map.image}}" alt="image">
+        {% endfor %}
+        </div><br>
+        <div class="d-flex p-2">
+        {% for graphic in graphic_list %}
+            <img class = 'img-informal'  src="http://{{graphic.image}}" alt="image">
+        {% endfor %}
+        </div>
+
+        <h2 class="heading">Actions Taken</h2>
+        {% for act_taken in actions_taken %}
+
+            {% for key, value in act_taken.items %}
+
+                {% if key == "organization_display" %}
+                
+                <p class="sub-heading">Actions Taken by {{value}}</p>
+                
+                {% endif %}
+
+                {% if key == 'summary' %}
+                
+                <p class="sub-heading">Descriptions</p>
+                
+                <p class="desc">{{value}}</p>
+        
+                {% endif %}
+
+                {% if key == "action_details" %}
+                <p class="sub-heading">Actions</p>
+                    {% for action in value %}
+                        {% for key, value in action.items %}
+                            {% if key == "name" %}
+                            <p class="actions">{{value}}</p>
+                            {% endif %}
+                        {% endfor %}
+                    {%endfor%}
+                {% endif %}
+
+            {% endfor %}
+            <hr>
+    {% endfor %}
+
+    <h2 class="heading">Resources</h2>
+    {% for resource in resources %}
+        {% for key, value in resource.items %}
+            {% if key == "date" %}
+            <p class="desc">{{value}}</p>
+            {%endif%}
+
+            {% if key == "source_description" %}
+            <p class="desc">{{value}}</p>
+            {%endif%}
+
+            {% if key == "url" %}
+            <a class="actions">{{value}}</a>
+            {%endif%}
+        {%endfor%}
```

Comment:
Why are we using this? Is normal lookup not available?
```suggestion
        <p class="desc">{{resource.date}}</p>
        <p class="desc">{{resource.source_description}}</p>
        <a class="actions">{{resource.url}}</a>
```

Reply:
resources is a m2m field so, normal lookup is not available

Reply:
hmm... what happens if you do as suggested above?

Reply:
here resources is a orderdict, so {{resource.date}} leads to an error 'collections.OrderedDict' object has no attribute 'date'. SO now i converted orderdict to normal dict to solve this issues. {{resource.date}} is working fine.

Reply:
Sounds good.

Make sure to update all usages related to this as well.